### PR TITLE
test run example notebooks

### PR DESCRIPTION
**Pull request by juanitorduz**
Closes https://github.com/py-econometrics/pyfixest/issues/606

### CODE REVIEW

File: scripts/run_notebooks.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+KERNEL_NAME: str = "python3"
+DOCS = Path("docs")
+NOTEBOOKS: list[Path] = [
+    # DOCS / "compare-fixest-pyfixest.ipynb",  # needs R
+    DOCS / "difference-in-differences.ipynb",
+    DOCS / "marginaleffects.ipynb",
+    DOCS / "quickstart.ipynb",
+    DOCS / "replicating-the-effect.ipynb",
+    DOCS / "stargazer.ipynb",  # failing notebook
```
Comment (juanitorduz) [inline scripts/run_notebooks.py:21]: @s3alfisc  in Cell[22] there is an error in the notebook (checked locally)

```python
NotImplementedError: <class 'pyfixest.estimation.feols_.Feols'>
```

See https://github.com/py-econometrics/pyfixest/actions/runs/10707033524/job/29686144381

File: scripts/run_notebooks.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from pathlib import Path
+
+import papermill
+from joblib import Parallel, delayed
+from tqdm import tqdm
+
+KERNEL_NAME: str = "python3"
+DOCS = Path("docs")
+NOTEBOOKS: list[Path] = [
+    # DOCS / "compare-fixest-pyfixest.ipynb",  # needs R
```
Comment (juanitorduz) [inline scripts/run_notebooks.py:16]: Let me work on this notebook on a different PR as it requires some investigation.

**Comment by juanitorduz** [comment]
@s3alfisc this is ready for review. This set up the tests. I commented two notebooks:

- One needs investigation (I need to install R locally hehe)
- The other notebook is failing because of `pyfixest` I think https://github.com/py-econometrics/pyfixest/pull/613#discussion_r1744227154

File: scripts/run_notebooks.py (hunk 2 above)
Comment (s3alfisc) [reply scripts/run_notebooks.py:16]: The error likely stems from the fact that Stargazer's default does not have pyfixest support - makes sense to handle this in another pr!

**Comment by juanitorduz** [comment]
ok! so any suggested changes or are we good to go @s3alfisc ?  :)

**Review by s3alfisc — APPROVED** [review]
Looks good to me, will merge now. Sorry that I did not finish the PR review yesterday, I got interrupted and then did not manage to get back to it. Btw I think you are successfully installing R via r2u, so the notebook depending on rpy2/fixest should execute properly =) thank you!

**Pull request merged**